frontend/src/metabase/models/components/ModelDetailPage/ModelDetailPage.tsx

```diff
@@ -55,12 +52,23 @@ function ModelDetailPage({
           onChangeName={onChangeName}
           onChangeCollection={onChangeCollection}
         />
-        <TabContent value={tab} onChange={setTab}>
-          <TabList
-            value={tab}
-            options={tabs}
-            onChange={tab => setTab(tab as ModelTab)}
-          />
+        <TabContent value={tab}>
+          <TabList>
+            <Tab.Link
```

Comment:
We use this `Component.Subcomponent` pattern only for styled-components that are used within the root component. Can we just use `TabLink` here, so it doesn't introduce another pattern?